### PR TITLE
Control panel actions (pause chat autoscroll, etc.)

### PR DESCRIPTION
**Pull request by RedFlames**
Adding some quality-of-life features to the control panel's chat panel:
image

I fiddled around with some stuff in `settingsCP()` in `dialog.js` because some checkbox & label attributes aren't fully confgurable with `mdcrd.checkbox` at the moment.

### CODE REVIEW

File: CelesteNet.Server.FrontendModule/Content/frontend/cp/js/components/dialog.js [hunk 1]
```diff
@@ -35,9 +40,15 @@ export class FrontendDialog {
 
       return el;
     }
-    const id = (id, gen) => el => {
+    const input = (id, gen, formid) => el => {
```
Comment (0x0ade) [inline CelesteNet.Server.FrontendModule/Content/frontend/cp/js/components/dialog.js:43]: Missing space between functions.

**Review by 0x0ade — CHANGES_REQUESTED** [review]
Looks good to me except for the indentation, spacing and input function order.

File: CelesteNet.Server.FrontendModule/Content/frontend/cp/js/components/dialog.js [hunk 2]
```diff
@@ -23,9 +23,14 @@ export class FrontendDialog {
...
   settingsCP() {
-    const row = (label, body) => el => rd$(el)`<span class="row"><span class="label">${label}</span><span class="body">${body}</span></span>`;
-    const group = (...items) => el => {
-      el = rd$(el)`<ul class="settings-group"></ul>`;
+    const row = (body) => el => rd$(el)`<span class="row">${body}</span>`;
+      const form = (id, body) => el => {
+          el = rd$(el)`<form>${body}</form>`;
+          el.id = id;
+          return el;
+      }
```
[the remark this answers is not in the document]
Comment (RedFlames) [reply CelesteNet.Server.FrontendModule/Content/frontend/cp/js/components/dialog.js]: Fixed.

File: CelesteNet.Server.FrontendModule/Content/frontend/cp/js/components/dialog.js [hunk 3]
```diff
@@ -47,9 +58,10 @@ export class FrontendDialog {
       title: "Settings: Control Panel",
       body: el => rd$(el)`
       <div>
-        ${group(
-          id("setting-sensitive", mdcrd.checkbox("Show sensitive data", s.sensitive))
-        )}
+        ${form("settings-form", fieldset(
+            row(input("setting-sensitive", mdcrd.checkbox("Show sensitive data", s.sensitive), "settings-form")),
+            row(input("setting-minimizeServerMsgs", mdcrd.checkbox("Collapse join messages (MOTD)", s.minimizeServerMsgs), "settings-form"))
```
[the remark this answers is not in the document]
Comment (RedFlames) [reply CelesteNet.Server.FrontendModule/Content/frontend/cp/js/components/dialog.js]: Also fixed.

File: CelesteNet.Server.FrontendModule/Content/frontend/cp/js/components/dialog.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (RedFlames) [reply CelesteNet.Server.FrontendModule/Content/frontend/cp/js/components/dialog.js]: Makes sense, I switched them around.

Comment (RedFlames) [reply CelesteNet.Server.FrontendModule/Content/frontend/cp/js/components/dialog.js]: Not sure I interpreted this correctly but I put blank lines between the function-variable-declarations? (idk what exactly to call them)

**Review by 0x0ade — APPROVED** [review]

**Pull request merged**